apps/marginfi-v2-ui/src/utils/actionBoxUtils.ts

```diff
@@ -571,3 +567,11 @@ export async function verifyJupTxSize(quoteResponse: QuoteResponse, connection:
 
   const txLength = tx.serialize().length;
 }
+
+export const debounceFn = (fn: Function, ms = 300) => {
```

Comment:
were you using useHooks previously? not an issue I just remember you mentioning their useDebounce.